### DISCUSSION TITLE
[Forge] How can I disable Fabric GUI reminder?

### DISCUSSION THREAD

**Discussion by gotoundo** [samolego/Taterzens#121] [Q&A] [answered]
Hello! Whenever someone interacts with one of my npcs, they get the text:
"GUIs are not supported in Forge version. Please install Fabric one."

We're happy without GUIs - is it possible to turn off this message? Thank you!

**Comment by samolego** [comment, marked as answer]
Hi, sorry to say that, but there's no way of disabling it, it was an oversight :(

**Reply by gotoundo** [reply]
It happens!! Thanks for the mod!
